enhancements/fbc-run-bundle.md

```diff
@@ -0,0 +1,268 @@
+
+- While initializing the FBC, should the channel received from the bundle annotations be used to initialize the default channel name in the `opm init` command?
+
+  - We should be able to handle bundles that don't have channel metadata on the annotations.
+
+- If we don’t get channel information from the bundle, then can we add it to a default channel or to the channel HEAD?
+
+- Can we assume the Channel’s “entries” field be initialized to an array with only one element, i.e., {name: “api-operator.v.0.1”} with no “replaces” field?
+
+  - "replaces" field is required for `run bundle-upgrade` but for run bundle, I think it's ok to assume for the channel entries to have just one entry.
+
+## Summary
+
+The goal of this enhancement is to discuss the impact of the new file based catalog (FBC) in Operator Lifecyle Manager and how it affects the Operator SDK, specifically the existing run-bundle command. This EP discusses the implementation/workflow changes of the `run bundle` command to support the new File-Based Catalog (FBC) index format along with continuing to support SQLite based index images.
+
+## Motivation
+
+One of the main goals of the Operator SDK is to enable operator developers to install and run their operators using Operator Lifecycle Manager (OLM). Currently, there are two commands in the Operator SDK for installing an operator bundle (`run bundle`) and upgrading an operator bundle to a newer version (`run bundle-upgrade`) using OLM. Operator SDK installs and upgrades the operator bundles using opm tooling that inherently uses a SQLite database under the hood, which is responsible for generating the registry database as well as the index images. We use `opm registry add` under the hood to create an ephemeral index image during the `run bundle(-upgrade)` workflow; however, that command is being deprecated.
+
+As OLM is transitioning to declarative config which is a more human readable and consumable format of a catalog that is in the form of `JSON/YAML`, Operator SDK should transition the existing `run bundle(-upgrade)` commands to support declarative config index images and default to using file based catalogs so that Operator authors can continue using Operator SDK to install/upgrade their operator bundles using the new FBC format.
+
+This proposal discusses changes to the `run bundle` functionality to be able to support file-based catalogs and how the registry pod should be created to support the generated FBC format i.e., to modify/replace the existing opm commands used to create the ephemeral index image during registry pod creation by leveraging the Operator Registry APIs.
+
+### Goals
+
+- Operator SDK’s `run bundle` command should work with file-based catalogs and be able to install an operator bundle using a valid FBC.
+
+- Operator SDK should create a valid file based catalog on the fly using the bundle image using Operator Registry APIs as a part of `run bundle` functionality.
```

Comment:
This seems like an internal implementation detail that does not necessarily need to be a goal. 
```suggestion
```